seacrowd/sea_datasets/thai_nner/thai_nner.py

```diff
@@ -0,0 +1,594 @@
+        "S-station",
+        "E-jargon",
+        "S-last",
+        "S-ocean",
+        "S-quantity",
+        "S-award",
+        "E-law",
+        "I-middle",
+        "E-firstname",
+        "E-facility:other",
+        "I-address",
+        "I-religion",
+        "S-time",
+        "E-periodic",
+        "S-vehicle",
+        "B-nickname",
+        "S-fold",
+        "B-role",
+        "E-rel",
+        "S-goverment",
+        "I-district",
+        "B-song",
+        "E-money",
+        "E-island",
+        "I-duration",
+        "E-electronics",
+        "B-sub_district",
+        "E-language",
+        "E-temperature",
+        "B-building",
+        "S-sports_event",
+        "E-concert",
+        "B-percent",
+        "S-island",
+        "I-museum",
+        "B-island",
+        "B-media",
+        "S-language",
+        "B-stock_exchange",
+        "S-title",
+        "E-natural_disaster",
+        "B-animal_species",
+        "B-facility:other",
+        "B-state",
+        "E-address",
+        "I-animate",
+        "S-stadium",
+        "I-province",
+        "B-city",
+        "B-army",
+        "I-weapon",
+        "B-middle",
+        "S-cardinal",
+        "E-date",
+        "S-event:others",
+        "S-periodic",
+        "I-speed",
+        "B-day",
+        "B-goverment",
+        "E-media",
+        "B-norp:others",
+        "S-nickname",
+        "E-district",
+        "I-natural_disaster",
+        "E-fold",
+        "I-language",
+        "S-rel",
+        "B-product:food",
+        "B-bridge",
+        "I-longtitude",
+        "B-orgcorp",
+        "B-award",
+        "E-nationality",
+        "E-weapon",
+        "E-province",
+        "I-tv_show",
+        "B-energy",
+        "I-building",
+        "B-roadname",
+        "B-continent",
+        "B-firstname",
+        "I-stock_exchange",
+        "S-norp:political",
+        "S-weight",
+        "S-restaurant",
+        "B-weapon",
+        "S-song",
+        "B-disease",
+        "E-river",
+    ]
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_seq_label",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema="seacrowd_seq_label",
+            subset_id=f"{_DATASETNAME}",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "thai_nner_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features({"index": datasets.Value("string"), "tokens": [datasets.Value("string")], "ner_tag": [datasets.Value("string")]})
+
+        elif self.config.schema == "seacrowd_seq_label":
+            features = schemas.seq_label_features(self.label_classes)
```

Comment:
Thank you for the clarification @bp-high . Since that's the case, could we revise the feature schema so the implementation satisfies the testcases?

```suggestion
            features = datasets.Features({"index": datasets.Value("string"), "tokens": [datasets.Value("string")], "ner_tag": [[datasets.Value("string")]]})
            
            # and for SEACrowd seq_label
```

same goes for the `SEACrowd` schema